moz_telemetry/modules/moz_telemetry/webrtc.lua

```diff
@@ -0,0 +1,60 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Mozilla Telemetry WebRTC Helper Module
+
+## Functions
+
+### filter_message
+
+Return true if the message contains a WebRTC payload, false otherwise.
+
+*Arguments*
+- none, called in the context of an output's process_message
+
+*Return*
+- bool (boolean) true if the current message contains a WebRTC payload
+
+--]]
+
+-- Imports
+local cjson = require "cjson"
+local type = type
+local read_message = read_message
+
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local function check_payload (payload)
+    if type(payload) ~= "table" then return false end
+    local w = payload["webrtc"] or {}
+    local i = w["IceCandidatesStats"] or {}
+    if next(i["webrtc"] or {}) or next(i["loop"] or {}) then
+        return true
+    end
+    return false
+end
+
+function filter_message ()
```

Comment:
Naming is a bit counter intuitive.  i.e. `if filter_message()`  to me true implies that the message should be filtered out (not kept)